libp2p/8-implementations.md

```diff
@@ -1,40 +1,57 @@
 8 Implementations
 =================
 
-This is a list of known libp2p module implementations. They are components that respect the interfaces and expectations defined in the "Interfaces" chapter, and can be composed to make a working libp2p library.
+An libp2p implementation should(recommended) follow a certain level of granulatiry when implementing different modules and functionalities, so that common interfaces are easy to expose, test and check for interoperability with other implementations.
+
+This is the list of current modules available for libp2p:
+
+- libp2p (entry point)
```

Comment:
It would be nice if each of these parts had a list of implementations in the different languages attached to it, like this:

```markdown
- abstract-transport [spec](link/to/spec) [go](link/to/go) [JS](link/to/js)
```